liquidator/detail/score_matrix_detail.h

```diff
@@ -79,10 +79,10 @@ scale(const PWM& pwm, const std::pair<double, double>& min_max, const unsigned r
 }
 
 // returns score; sequences with invalid characters return 0.
-unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
-               const std::string& sequence,
-               const size_t begin,
-               const size_t end)
+inline unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
```

Comment:
Here is some interesting discussion (with references to the standard) on inline vs static inline:

http://stackoverflow.com/a/4193698/1007353
http://stackoverflow.com/a/189162/1007353